### DISCUSSION TITLE
Questions on tidying up the code base

### DISCUSSION THREAD

**Discussion by ghost** [alpinejs/alpine#3434] [4. General]
I was looking through the code base, and there were a few questions that came up initially. I wanted to check with the maintainers before submitting any of these pull requests in case I was missing anything.

1. `index.html`, `index2.html`, and `morph.html` appear to be playground files. Thoughts on removing these files?
2. Jest tests appear to be failing, and aren't run in CI. Are there plans to fix these, migrate them to Cypress, or remove them?
3. `alpinejs` has a dependency on `@vue/reactivity`, but that dependency is also (from what I see in the build output) bundled with `alpinejs`. When installing `alpinejs` with `npm`, for example, I get three dependencies rather than one. Should `@vue/reactivity` be made a dev dependency?
4. Similar to question three, should `nprogress` be made a dev dependency of `@alpinejs/navigate`?

**Comment by ekwoka** [comment]
I would be on the side of not prebundling for the non-cdn builds.

**Reply by ghost** [reply]
That works! Thanks for the feedback.
